src/Checkbox/Checkbox.less

```diff
@@ -2,22 +2,73 @@
 .moon-checkbox {
 	cursor: pointer;
 
+	.moon-formcheckbox-item &,
+	.moon-toggle-item & {
+		.moon-icon {
+			visibility: hidden;
+			background-color: transparent;
+		}
+		&[checked] .moon-icon {
+			visibility: visible;
+
+			.moon-neutral & {
+				background-color: transparent;
+			}
+		}
+	}
+
 	.moon-icon {
-		visibility: hidden;
 		margin: 0;
 		padding: 0;
+		width: 30px;
+		height: 30px;
+		line-height: 30px;
+		font-size: 60px;
+		border-radius: 15px;
+		color: transparent;
+		background-color: @moon-dark-gray;
```

Comment:
It's best practice to not refer to these variables outside of the `moonstone-dark/light.less` files, and instead assign the `@moon-dark-gray` to a more descriptive color variable name, like `@moon-checkbox-bg-color: @moon-dark-gray;`. This is simpler and easier to maintain over time and abstracts the color value from the component LESS files. Be sure to update both the light and dark color files.